packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -566,10 +603,18 @@ class StatefulGraphChatListClient implements StatefulClient<GraphChatListClient>
       draft.status = chatThreads.length > 0 ? 'chats loaded' : 'no chats';
       draft.chatThreads = chatThreads;
       draft.moreChatThreadsToLoad = nextLink !== undefined && nextLink !== '';
-      if (this._initialSelectedChatId) {
-        draft.internalSelectedChat = chatThreads.filter(c => c.id === this._initialSelectedChatId)[0];
-        this._initialSelectedChatId = undefined;
-        draft.initialSelectedChatSet = true;
+      draft.fireOnSelected = false;
+      if (this._initialSelectedChatId && this._initialSelectedChatId !== '') {
```

Comment:
First condition covers the empty string in the second condition.

Reply:
I'm seeing this done in several places though, so, for consistency, I'm also okay with keeping it.

Reply:
Actually, I am in favor of removing it. This seems like it might be confusing given that constraint is already checked.

Reply:
I have updated all locations, please review and let me know if you see any where I missed.